src/utils/configure-nuxt.ts

```diff
@@ -54,29 +54,42 @@ export function configureNuxt(
   })
 
   nuxt.hook('prepare:types', ({ references }) => {
-    references.push({ types: 'vuetify' })
+    // references.push({ types: 'vuetify' })
     references.push({ types: 'vuetify-nuxt-module/custom-configuration' })
     references.push({ types: 'vuetify-nuxt-module/configuration' })
     references.push({ path: ctx.resolver.resolve(runtimeDir, 'plugins/types') })
   })
 
-  /* nuxt.hook('components:extend', async (c) => {
-    const components = await ctx.componentsPromise
-    Object.keys(components).forEach((component) => {
-      c.push({
-        pascalName: component,
-        kebabName: toKebabCase(component),
-        export: component,
-        filePath: 'vuetify/components',
-        shortPath: 'vuetify/components',
-        chunkName: toKebabCase(component),
-        prefetch: false,
-        preload: false,
-        global: false,
-        mode: 'all',
-      })
+  nuxt.hook('components:extend', async (c) => {
+    // components/XXXX
+    // labs/XXX
+    const [components, labs] = await Promise.all([
+      ctx.componentsPromise,
+      ctx.labComponentsPromise,
+    ])
+    Object.entries(labs).forEach(([component, entry]) => {
+      components[component] = entry
     })
-  }) */
+    Object
+      .entries(components)
+      .forEach(([component, entry]) => {
+        const from = entry.from
+        const name = from.split('/').at(-1)
+        const filePath = `vuetify/lib/${entry.from}`
+        c.push({
+          pascalName: component,
+          kebabName: toKebabCase(component),
+          export: component,
+          filePath,
+          shortPath: filePath,
```

Comment:
maybe we should try with `vuetify/components`, `vuetify` resolving to `lib/index.d.mts`:

![imagen](https://github.com/user-attachments/assets/8ce9c255-000d-4af0-a06d-967c621557a7)